_posts/0000-01-02-vbaofficesolutionsllc.md

```diff
@@ -0,0 +1,3 @@
+layout:  slide
```

Comment:
what the hell is this for?

Reply:
just typing shit because the space is here and don't know what else to do with it...